lib/view_component_contrib/railtie.rb

```diff
@@ -2,11 +2,25 @@
 
 module ViewComponentContrib
   class Railtie < Rails::Railtie
-    config.view_component.preview_paths << File.join(ViewComponentContrib::APP_PATH, "views")
+    if config.view_component.respond_to?(:previews)
```

Comment:
I think, it should be `if config.view_component.previews.respond_to?(:paths)` or `if !config.view_component.previews.nil?`.

I guess, the `config.view_component` object is an instance of ActiveSupport::Options, and it acts like a Hashie (a Hash with key accessor methods).